src/blocks/Saawn - cartAnim/script.js

```diff
@@ -0,0 +1,50 @@
+const button = document.querySelector("button");
+let timer;
+let timer2;
+const audio = new Audio("https://lasonotheque.org/UPLOAD/mp3/1417.mp3");
```

Comment:
License please, see other pull request by you.

Reply:
licence ? of what, audio?

https://lasonotheque.org/ 

audio from here are free and royalty free to use
![Screenshot (8)](https://user-images.githubusercontent.com/58899511/197382050-b963abb6-64c6-4c12-99c4-1613d1b233d2.png)